cmd/subnetcmd/join.go

```diff
@@ -704,7 +683,18 @@ func promptStakeAmount(subnetName string, isValidator bool, network models.Netwo
 	return initialSupply, nil
 }
 
-func printJoinCmd(subnetID string, networkID string, vmPath string) {
+func printJoinCmd(subnetID string, network models.Network, vmPath string) {
+	networkIDValue := ""
+	switch network.Kind {
```

Comment:
this should be part of the network struct like network.Tag() or something like that

Reply:
so it would look like `ntagetworkIDValue := network.Tag()`

Reply:
maybe better name